felles/src/main/kotlin/no/nav/familie/kontrakter/felles/dokarkiv/Dokumenttype.kt

```diff
@@ -18,6 +18,7 @@ enum class Dokumenttype {
     BARNETRYGD_UTVIDET,
     BARNETRYGD_VARSEL_OM_REVURDERING,
     BARNETRYGD_VARSEL_OM_REVURDERING_DELT_BOSTED_PARAGRAF_14,
+    BARNETRYGD_VARSEL_OM_REVURDERING_SAMBOER,
```

Comment:
Har du lagt til metadata for denne?

Reply:
Sånn som for denne, ellers tror jeg vi får feil i prod når vi prøver å bruke det
https://github.com/navikt/familie-integrasjoner/pull/576

Reply:
Man legger vel til metadata i familie-integrasjoner? Og det må vel skje etter at denne er merget

Reply:
Jaa selvfølgelig!